packages/api/test/search.test.ts

```diff
@@ -0,0 +1,107 @@
+import type { inferProcedureInput } from "@trpc/server";
+import { describe, expect, it } from "vitest";
+
+import type { AppRouter } from "../src/root";
+import { getCaller } from "./helpers";
+
+describe("Search route", async () => {
+  const caller = await getCaller();
+  type Input = inferProcedureInput<AppRouter["search"]["byTerm"]>;
```

Comment:
Let's move this outside. Type declarations should be at the beginning of the file